CyclopsNuclearReactor/CyNukeRodSaveData.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace CyclopsNuclearReactor
+{
+    using Common.EasyMarkup;
+    using System.Collections.Generic;
+
+    internal class CyNukeRodSaveData : EmPropertyCollection
```

Comment:
There is ambiguity between the `EmPropertyCollection` defined as part of the common project and the one exposed by `MoreCyclopsUpgrades` which is now a reference.  
It might make sense to make all the `EmProperty` classes `internal`.

Reply:
That didn't work.

Reply:
Turns out, it wasn't working because I was using a DLL reference.  
Now that the update has been released as MoreCyclopsUpgrades 3.1.1, the ambiguity errors are gone.